### PR TITLE
SCW-390: update aa with astar support

### PR DESCRIPTION
**Pull request by maxisch**
## Description

Add Astar support to AA SDK

## Test plan

Manual testing

### CODE REVIEW

File: packages/client/wallets/aa/src/utils/constants.ts
Comment (maxisch) [inline packages/client/wallets/aa/src/utils/constants.ts:16]: Blocked by zerodev. Context: https://crossmint.slack.com/archives/C052QC5JGSD/p1709180839028699

File: packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts
Comment (afeight) [inline packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts:112]: very nice 🙏

File: packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts
Comment (afeight) [inline packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts:106]: this says optimism?

Comment (afeight) [reply packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts:106]: hmm looks like it may have been https://testnet.explorer.zora.energy/ but is now not. If we do null here that's fine, it's probably no longer in existence

File: packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts
Comment (afeight) [inline packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts:114]: did this already exist elsewhere?

**Review by afeight — APPROVED** [review]
cool, approved pending release of base 0.0.7 and zerodev support

**Pull request merged**
